plugins/hydra_ray_launcher/pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = -p no:warnings
```

Comment:
do we still have warnings now that Ray finally released 1.0?

Reply:
warnings still exists :(